src/main/java/duke/task/Todo.java

```diff
@@ -0,0 +1,41 @@
+package duke.task;
+
+import duke.exception.EmptyDescriptionException;
+
+public class Todo extends Task {
+    public Todo(String description) {
+        super(description);
+    }
+
+    /**
+     * Initialize a new Todo from user input.
+     *
+     * @param input User input.
+     * @return A new Todo object.
+     * @throws EmptyDescriptionException If todo description is empty.
+     */
+    public static Todo initializeFromInput(String input) throws EmptyDescriptionException {
```

Comment:
I like how you create an individual method to construct a new ToDo from user input.👍 I believe it makes the interaction between classes clearer and easier to handle.